### ISSUE TITLE
Bug: variable overflow in peak column of narrowPeak file

### ISSUE THREAD

**Issue by diego-rt** [macs3-project/MACS#500]
**Describe the bug**
Hi, 
for some reason we are getting negative values in the peak column of the narrowPeak file from our data (see image attached). This is probably due to the size of our chromosomes (32 gbp genome) and smells to me like a variable overflow when computing the peak summits.

**To Reproduce**
macs3 callpeak -f BAMPE -t ${dedupBAM} -g 2.0e10 -n \${name} -B --SPMR --seed 23 -q 0.01 --call-summits --keep-dup all

**Expected behavior**
I believe these are supposed to be positive values.

**Screenshots**
<img width="932" alt="Screenshot 2022-02-01 at 20 11 22" src="https://user-images.githubusercontent.com/80003172/152035460-f78b9fae-ecd3-4d5d-af9e-a33cf4907ed9.png">

**System (please complete the following information):**
Linux, macs3a6

**Comment by taoliu** [comment]
Right... It's an overflow. Could you share with me the first ten rows of the narrowPeak file from MACS?

**Comment by taoliu** [comment]
@diego-rt The first ten rows seem fine to me. Will you see negative values in the last column of the narrowPeak file? Also, the chromosome names (C00????) seem different from the message from R (e.g. chr10p). 

Regarding HMMRATAC, we are working on implementing it in MACS3 with Python in a separate branch right now. Thanks for the information of the CSI index! We will work on that.

**Comment by diego-rt** [comment]
Apologies for the delayed reply! 

Oh that's great to hear about HMMRATAC! Do you have an estimated release timeframe in mind?

Yes, the chromosome names are different becase some are chromosome scale scaffolds and some are unplaced contigs (CXXXXXXX), but they are all part of the same genome reference: Axolotl genome v6.

Here go the lowest 1500 scores on the 10th column:

[macs3_Debug_output.txt](https://github.com/macs3-project/MACS/files/8023886/macs3_Debug_output.txt)
